chart/templates/aws-load-balancer-controller.yaml

```diff
@@ -20,6 +20,13 @@ spec:
             name: {{ .Values.awsLoadBalancerController.serviceAccountName }}
             create: false
         {{- toYaml .Values.awsLoadBalancerController | nindent 10 }}
+      parameters:
+      - name: aws-load-balancer-controller.clusterName
```

Comment:
why do we need these here, they are already in the values above. it looks like the only one we need to add is `region`, no?

Reply:
I was sleepwalking. How about this time? I have already checked the operation.